README.md

```diff
@@ -2,55 +2,63 @@
 
 # Project Name
 
-> Description the project.
+> Todo is a web application that makes it easier for users to organize their chores while working on any project. The board displays all of the activities that need to be done, as well as the ability to delete completed tasks from the board.
 
 
 ## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+- To get a local copy up and running follow these simple example steps
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+### Technologies
+- HTML
+- CSS
+- JS
+- Git
+- Webpack
+
 ### Setup
+Cloning the project
+> git clone ...............
```

Comment:
- [OPTIONAL] Please it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non technical users. Please add descriptive steps from cloning your app, to seeing the results displayed on the browser.